### PR TITLE
Fix error handling related to nearmint changes

### PR DESCRIPTION
**Pull request by vgrichina**

### CODE REVIEW

File: nearlib/nearclient.js [hunk 1]
```diff
@@ -37,22 +37,17 @@ class NearClient {
...
-            logs = response.log.split('\n');
-        }
-        logs.forEach(line => {
-            console.log(`[${contractAccountId}]: ${line}`);
-        });
-        // If error, raise exception after printing logs.
-        const code = response.code || 0;
-        if (code != 0) {
-            throw Error(response.info);
+        const ignoreErrors = true;
```
Comment (janedegtiareva) [inline nearlib/nearclient.js]: Could you leave a comment somewhere about what this does and why we need this?

File: nearlib/nearclient.js [hunk 2]
```diff
@@ -37,22 +37,17 @@ class NearClient {
...
+            return json;
+        } catch(e) {
+            _printLogs(contractAccountId, e.log);
+            throw e;
         }
-        const json = JSON.parse(_base64ToBuffer(response.value).toString());
-        return json;
     }
 
     async getTransactionStatus(transactionHash) {
```
Comment (janedegtiareva) [inline nearlib/nearclient.js]: does this need a doc? I'd like to definitely see some common status codes explained.

File: nearlib/nearclient.js (hunk 1 above)
Comment (vgrichina) [reply nearlib/nearclient.js]: If you mean `ignoreErrors` it just needs to be removed, accidental leftover

File: nearlib/nearclient.js (hunk 2 above)
Comment (vgrichina) [reply nearlib/nearclient.js]: none of the methods in this class are currently documented. Probably not a good time to add comment, given this is somewhat unrelated bug fix.

**Comment by vgrichina** [comment]
Ported to https://github.com/nearprotocol/nearlib/pull/4

**Pull request closed without merging**